pyproject.toml

```diff
@@ -61,25 +63,25 @@ gdspy = {version="*", optional = true}
 gdstk = [
     {version="*", optional = true, platform = "linux"},
     {version="*", optional = true, platform = "win32"},
-    {version="<0.9.38", optional = true, platform = "darwin"},
+    {version=">=0.9.49", optional = true, platform = "darwin"},
 ]
 
 # jax
 jaxlib = [
     {version = ">=0.4.1,<=0.4.14", platform = "linux",  source="jaxsource", optional = true},
     {version = ">=0.4.1,<=0.4.14", platform = "darwin",  source="jaxsource", optional = true},
-    {version = ">=0.4.13,<=0.4.14", platform = "win32", source="jaxsource", optional = true}
+    {version = ">=0.4.13,<=0.4.14", platform = "win32", source="jaxsource",python = "^3.9", optional = true}
 ]
 jax = [
     {version = ">=0.4.1,<=0.4.14", extras = ["cpu"], platform = "linux", source="jaxsource", optional = true},
     {version = ">=0.4.1,<=0.4.14", extras = ["cpu"], platform = "darwin",  source="jaxsource", optional = true},
-    {version = ">=0.4.13,<=0.4.14", extras = ["cpu"], platform = "win32", source="jaxsource", optional = true}
+    {version = ">=0.4.13,<=0.4.14", extras = ["cpu"], platform = "win32", python = "^3.9", source="jaxsource", optional = true}
```

Comment:
we're already excluding any windows < 3.9 now? so maybe we dont need this `python` condition anymore?

Reply:
So I think we're excluding it out of tests, I thought we were still supporting 3.8 where this would have to be included in the packaging?

Reply:
hm, I thought we were excluding windows with python 3.8 because jax didn't work at all? or was it just that we couldn't install it in our automatic test? feel free to keep it how you had it, maybe I just misunderstood

Reply:
I think it was because it was badly packaged for win py3.8. So I think the condition above just excludes that if I've understood the poetry requirements well.